wgpu-core/src/command/bind.rs

```diff
@@ -200,13 +200,17 @@ mod compat {
                 entries: (0..hal::MAX_BIND_GROUPS).map(|_| Entry::empty()).collect(),
             }
         }
-        fn make_range(&self, start_index: usize) -> Range<usize> {
+
+        pub fn nr_of_valid_entries(&self) -> usize {
```

Comment:
note: I split the introduction of `BoundBindGroupLayouts::nr_of_valid_entries` out into d93deb0a.